docs/sdk/push/migration/xamarin-android.md

```diff
@@ -14,9 +14,9 @@ ms.tgt_pltfrm: android
 
 [!include[](introduction-android.md)]
 
-## 1. Mono version
+## 1. Xamarin.Android version
 
-If your Android project does not target Mono framework version 8.1 or higher, you will not be able to update the Push package.
+If your Android project does not target Xamarin.Android version 8.1 or higher, you will not be able to update the Push package.
```

Comment:
If the target framework version of your Android application is lower than 8.1, you will not be able to update the Push package.